executor/aggfuncs/window_func_test.go

```diff
@@ -207,3 +207,17 @@ func (s *testSuite) TestMemRank(c *C) {
 		s.testWindowAggMemFunc(c, test)
 	}
 }
+
+func (s *testSuite) TestMemCumeDist(c *C) {
```

Comment:
IMO, moving the `TestMemCumeDist` to the `func_cume_dist_test.go` is better. Note that we should add an open source license to a new file.

Reply:
Ok, should I move the relation part in `TestWindowFunctions` to the new `func_cume_dist_test.go` file too?

Reply:
just `TestWindowFunctions` is ok.

Reply:
done.